utilities/test_suite/HOST/Tensor_host.cpp

```diff
@@ -1594,26 +1593,26 @@ int main(int argc, char **argv)
                 }
 
                 // print reduction functions output array based on different bit depths, and precision desired
-                int precision = ((dstDescPtr->dataType == RpptDataType::F32) || (dstDescPtr->dataType == RpptDataType::F16) || testCase == 90 || testCase == 91) ? 3 : 0;
-                if (dstDescPtr->dataType == RpptDataType::F32 || testCase == 90 || testCase == 91)
+                int precision = ((dstDescPtr->dataType == RpptDataType::F32) || (dstDescPtr->dataType == RpptDataType::F16) || testCase == TENSOR_MEAN || testCase == TENSOR_STDDEV) ? 3 : 0;
+                if (dstDescPtr->dataType == RpptDataType::F32 || testCase == TENSOR_MEAN || testCase == TENSOR_STDDEV)
                     print_array(static_cast<Rpp32f *>(reductionFuncResultArr), reductionFuncResultArrLength, precision);
                 else if (dstDescPtr->dataType == RpptDataType::U8)
                 {
-                    if (testCase == 87)
+                    if (testCase == TENSOR_SUM)
```

Comment:
Can this be done even for the switch case statement instead of saying case 0 / 1 / 2?